### PR TITLE
CORE - Application importer uses the Kratos Logger to print

### PR DESCRIPTION
**Pull request by maceligueta**
I used the empty label.

### CODE REVIEW

**Review by roigcarlo — COMMENTED** [review]
Ok, but if a lot of people is gonna use the logger like that maybe we should start thinking to provide a way to print so the ":" does not appear in the output... @pooyan-dadvand ¿

**Comment by maceligueta** [comment]
Related to #1928

File: kratos/python_interface/application_importer.py
Comment (philbucher) [inline kratos/python_interface/application_importer.py]: shouldn't this be `PrintInfo?`

Comment (maceligueta) [reply kratos/python_interface/application_importer.py]: Thanks @philbucher . I just changed it and pushed.

**Comment by maceligueta** [comment]
Can we merge this? Needs approval.

**Review by roigcarlo — APPROVED** [review]
Ok to me 👍 but I've done changes related to this, someone else should take a look

**Review by pooyan-dadvand — APPROVED** [review]
👍

**Pull request merged**
